### PR TITLE
Update

### PR DESCRIPTION
**Pull request by TollerNamen**
….

### CODE REVIEW

**Comment by TollerNamen** [comment]
oh wait i have made my first pr

**Comment by TollerNamen** [comment]
well i have initialied the variable „level“ in rank.py before making a pic while subtracting it by one which is the point because it confuses me that the levels in the rank cmd are not equal to the levels displayed on the leaderboard cmd. Dont know if that works though

**Comment by TollerNamen** [comment]
ive been told to make pr in discord btw

File: bot/commands/levels/rank.py [hunk 1]
```diff
@@ -87,11 +87,12 @@ async def execute(self, arguments, message) -> None:
                 rank = i + 1
             bg_image = await self.get_bg(user.id)
             pic = None
+            level=result[1] - 1
             try:
                 pic = await generate_profile(
                     bg_image=bg_image if bg_image else None,
                     profile_image=user.avatar.url,
-                    level=result[1],
+                    level,
```
Comment (logicguy1) [inline bot/commands/levels/rank.py:95]: You still need level=level, please run your code before submitting a PR

Comment (timelessnesses) [reply bot/commands/levels/rank.py:95]: why is it minus one i may ask?

File: bot/commands/levels/rank.py [hunk 2]
```diff
@@ -103,7 +104,7 @@ async def execute(self, arguments, message) -> None:
...
```
Comment (timelessnesses) [inline bot/commands/levels/rank.py:107]: this too

**Review by timelessnesses — COMMENTED** [review]
quite concerning pr since why would you minus the level number by one

**Comment by TollerNamen** [comment]
> quite concerning pr since why would you minus the level number by one

To get equal results with the leaderboard cmd.

**Comment by demirdegerli** [comment]
Should be a better solution

**Pull request closed without merging**